_posts/2019-09-09-what-is-mtls.md

```diff
@@ -4,25 +4,63 @@ current: post
+First of all, mTLS is configurable. It is up to the server and client to decide which certificates are considered 'trustworthy.' In the examples that we provided, we set up the system in a way to trust only WoTT provided certificates; but you absolutely can modify this to include other CA provided certifcates. It is up to the developer's discretion. 
+
+But how does it actually work? We've mentioned already the 'handshake' as a concept, but this is actually the name given to the technology TLS uses. The TLS handshake protocol can be found [here](https://www.ietf.org/rfc/rfc5246.txt) but it's long and wordy. What's important to know is that the TLS protocol is part of a layered structure and is sat between the transport (also known as TCP) and application layers.
+
+{% asset blog/tls-structure.png srcset:width="1300 2x" srcset:width="650 1x" alt="Breakdown of the TLS structure" %}
+
+The protocol/handshake layer is where the cryptographic 'magic' takes place. It manages the encryption algorithm used (also known as cipher) and key exchange, usually known as PKI (public key infrastructure). The PKI is signed for by the Certificate Authority and is what gives an endpoint its cryptographic identity. Basically, an x509 certificate is generated by this handshake protocol based off of the technologies each endpoint is using and then these certificates verified by the appropriate CA. With mTLS, it's the exact same except both the server and client verify each other. You can think of the CA as a signiature vouching for a device's identity.
+
+## Why you should care?
+
+For a start, it's probably the most common thing used since SSL became outdated. Several companies use mTLS. For example Skype uses it to secure business servers; and Cloudflare has options to employ mTLS. It's particularly important in cases where you want encrypted, secure traffic in *both* directions. Many devices now do not use logins (such as IoT devices) meaning that they would either be blocked from accessing a resource, or access a resource insecurely. Using mTLS is a way to provide an identity to a device/server that can be cryptographically verified. In other words, it's a way of making your resources more flexible and available whilst still being secure.
+
+It's also a good way for you to secure yourself if you intend on being a server provider. Much of how internet security is done currently is verifying a server from a given client, it's a lot harder to protect yourself as the server. Usernames and passwords are an option, but they are exploitable and unreliable. More and more there is a need to move away from such technologies and start using cryptographic signage to identify trustworthy devices. Of course, no technology is perfect. TLS is updated frequently to counter this and it is the developer's responsibility to choose trusted CAs. For now, mTLS is a strong, lightweight method of securing your server and client endpoints.
+
+## <a name = "conclusions"> </a> Conclusions
```

Comment:
Same here.

Reply:
are you sure? These work on my machine and I have this in previous articles too

Reply:
It might work, but it's not valid HTML. It's probably a forgiving browser.

Reply:
it's markdown

Reply:
No it isn't. `<a name = "conclusions"> </a>` is HTML. You can (often) include HTML in Markdown.